personal.bos/include/personal.bos/personal.bos.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ *  @file
+ *  @copyright defined in eos/LICENSE.txt
+ */
+#pragma once
+
+#include <eosiolib/eosio.hpp>
+
+#include <string>
+
+namespace eosio {
+
+   using std::string;
+
+   class [[eosio::contract("personal.bos")]] personal_contract : public contract {
+      public:
+         using contract::contract;
+
+         [[eosio::action]]
+         void setpersonal( name account, string key, string value);
+
+         [[eosio::action]]
+         void sethomepage( name account, string url);
+
+         string get_personal( name account, string key ){
```

Comment:
name key  ->std::string key

Reply:
change the type of key from string to name